src/sequelize/seeders/6_user_seed.js

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const bcrypt = require('bcrypt');
+require('dotenv/config');
+
+const salt = bcrypt.genSaltSync(10);
+const { SEEDER_ADMIN_PASSWORD, SEEDER_SUPER_ADMIN_PASSWORD } = process.env;
+const adminPassword = bcrypt.hashSync(SEEDER_ADMIN_PASSWORD, salt);
```

Comment:
what do you think by adding @harerakalex 's utils file and you use his reusable functions for hashing the password? 
This is optional, you can even leave it like that then someone who will rebase after his PR got merged can change this.

Reply:
Since its impossible for us to use typescript or es6 in sequelize seeders, its not possible to export functions and import them in es5 modules in this case. Hence I am leaving this as it is. Unless there is another solution I am not aware of